config/templates/cloud-config-controller

```diff
@@ -408,14 +358,241 @@ write_files:
+              - name: var-run-calico
+                hostPath:
+                  path: /var/run/calico
+              - name: cni-bin-dir
+                hostPath:
+                  path: /opt/cni/bin
+              - name: cni-net-dir
+                hostPath:
+                  path: /etc/kubernetes/cni/net.d
+              - name: etcd-certs
+                secret:
+                  secretName: calico-etcd-secrets
+              - name: dns
+                hostPath:
+                  path: /etc/resolv.conf
+
+      ---
+
+      apiVersion: extensions/v1beta1
+      kind: Deployment 
+      metadata:
+        name: calico-policy-controller
+        namespace: kube-system
+        labels:
+          k8s-app: calico-policy
+        annotations:
+          scheduler.alpha.kubernetes.io/critical-pod: ''
+          scheduler.alpha.kubernetes.io/tolerations: |
+            [{"key": "node.alpha.kubernetes.io/role", "value": "master", "effect": "NoSchedule" },
+            {"key":"CriticalAddonsOnly", "operator":"Exists"}]
+
+      spec:
+        replicas: 1
+        template:
+          metadata:
+            name: calico-policy-controller
+            namespace: kube-system
+            labels:
+              k8s-app: calico-policy
+          spec:
+            hostNetwork: true
+            containers:
+              - name: calico-policy-controller
+                image: calico/kube-policy-controller:v0.5.2
+                env:
+                  - name: ETCD_ENDPOINTS
+                    valueFrom:
+                      configMapKeyRef:
+                        name: calico-config
+                        key: etcd_endpoints
+                  - name: ETCD_CA_CERT_FILE
+                    valueFrom:
+                      configMapKeyRef:
+                        name: calico-config
+                        key: etcd_ca
+                  - name: ETCD_KEY_FILE
+                    valueFrom:
+                      configMapKeyRef:
+                        name: calico-config
+                        key: etcd_key
+                  - name: ETCD_CERT_FILE
+                    valueFrom:
+                      configMapKeyRef:
+                        name: calico-config
+                        key: etcd_cert
+                  - name: K8S_API
+                    value: "https://kubernetes.default:443"
+                  - name: CONFIGURE_ETC_HOSTS
+                    value: "true"
+                volumeMounts:
+                  - mountPath: /calico-secrets
+                    name: etcd-certs
+            volumes:
+              - name: etcd-certs
+                secret:
+                  secretName: calico-etcd-secrets
+  
+  - path: /opt/bin/populate-tls-calico-etcd
     owner: root:root
+    permissions: 0700
     content: |
       #!/bin/bash -e
-      /usr/bin/curl -H "Content-Type: application/json" -XPOST --data-binary @"/srv/kubernetes/manifests/calico-system.json" "http://127.0.0.1:8080/api/v1/namespaces"
```

Comment:
looks like `/srv/kubernetes/manifests/calico-system.json` is left in userdata and not use anywhere